### PR TITLE
feat(header): create header UI

### PR DESCRIPTION
**Pull request by mthanh2209**

### CODE REVIEW

File: javascript-practice/src/index.html [hunk 1]
```diff
@@ -10,8 +10,53 @@
...
+				<img class="menu-icon" src="assets/icons/menu.svg" alt="" />
+				<div class="menu-toggle">
+					<img class="cart-icon" src="assets/icons/shopping-cart.svg" alt="" />
+					<img class="avatar-icon" src="assets/icons/user-avatar.svg" alt="" />
+				</div>
+			</div>
+		</div>
+		<div class="line"></div>
+		<nav>
+			<!-- test API -->
```
Comment (hungnguyenhuu-agilityio) [inline javascript-practice/src/index.html:34]: What is this comment? Should not appear here

File: javascript-practice/src/index.html [hunk 2]
```diff
@@ -10,8 +10,53 @@
...
+			<div class="nav-menu">
+				<h1 class="menu-text">Welcome to furniture website</h1>
+				<img class="menu-icon" src="assets/icons/menu.svg" alt="" />
+				<div class="menu-toggle">
+					<img class="cart-icon" src="assets/icons/shopping-cart.svg" alt="" />
+					<img class="avatar-icon" src="assets/icons/user-avatar.svg" alt="" />
+				</div>
+			</div>
+		</div>
+		<div class="line"></div>
```
Comment (hungnguyenhuu-agilityio) [inline javascript-practice/src/index.html:32]: class naming

File: javascript-practice/src/index.html [hunk 3]
```diff
@@ -10,8 +10,53 @@
 
 <body>
 	<!-- START HEADER SECTION -->
-	<header></header>
-	<!-- END HEADER SECTION -->
+	<header class="header">
+		<div class="top-nav">
+			<span class="header-search">
+				<img class="search-icon" src="assets/icons/search.svg" alt="" />
+				<img class="close-icon" src="assets/icons/close.svg" alt="">
```
Comment (hungnguyenhuu-agilityio) [inline javascript-practice/src/index.html:17]: Where is alt? I do not think that, we will have 2 images here. Please check again

File: javascript-practice/src/index.html [hunk 4]
```diff
@@ -10,8 +10,53 @@
 
 <body>
 	<!-- START HEADER SECTION -->
-	<header></header>
-	<!-- END HEADER SECTION -->
+	<header class="header">
+		<div class="top-nav">
+			<span class="header-search">
+				<img class="search-icon" src="assets/icons/search.svg" alt="icon search" />
+				<img class="close-icon" src="assets/icons/close.svg" alt="icon close" />
```
Comment (hungnguyenhuu-agilityio) [inline javascript-practice/src/index.html:17]: Why you have 2 images in a span? So confused, could you help to check it again? @mthanh2209

File: javascript-practice/src/index.html [hunk 5]
```diff
@@ -38,6 +83,34 @@
...
+		}
+
+		let menu = document.querySelector('.menu-toggle')
+		let menuEl = document.querySelector('.menu-icon')
+		let header = document.querySelector('header')
+
+		menuEl.onclick = () => {
+			header.classList.toggle('open')
+		}
+	</script>
```
Comment (hungnguyenhuu-agilityio) [inline javascript-practice/src/index.html:113]: Why you have this scripts block here?

File: javascript-practice/src/index.html [hunk 6]
```diff
@@ -10,8 +10,53 @@
 
 <body>
 	<!-- START HEADER SECTION -->
-	<header></header>
-	<!-- END HEADER SECTION -->
+	<header class="header">
```
Comment (hungnguyenhuu-agilityio) [inline javascript-practice/src/index.html:13]: naming

File: javascript-practice/src/index.html (hunk 4 above)
Comment (mthanh2209) [reply javascript-practice/src/index.html:17]: Thanks. I fixed it.

File: javascript-practice/src/index.html (hunk 5 above)
Comment (mthanh2209) [reply javascript-practice/src/index.html]: Sorry, I mistook and I updated it

File: javascript-practice/src/index.html (hunk 6 above)
Comment (mthanh2209) [reply javascript-practice/src/index.html]: Thanks, i updated

**Pull request merged**